### PR TITLE
fixed custom option array converter option fields data

### PR DESCRIPTION
**Pull request by LorensDomins**

### CODE REVIEW

**Review by mkucmus — APPROVED** [review]
great, now the options are cast and have proper values :+1:

File: src/module-vsbridge-indexer-catalog/ArrayConverter/Product/CustomOptionConverter.php [hunk 1]
```diff
@@ -92,8 +92,8 @@ private function filterData(array $option): array
     {
         $option = $this->dataFilter->execute($option, $this->fieldsToDelete);
         $option['sort_order'] = (int) $option['sort_order'];
-        $option['option_id'] = (int) $option['sort_order'];
-        $option['option_type_id'] = (int) $option['sort_order'];
+        $option['option_id'] = (int) $option['option_id'];
+        $option['option_type_id'] = (int) $option['option_type_id'];
```
Comment (revlis-x) [inline src/module-vsbridge-indexer-catalog/ArrayConverter/Product/CustomOptionConverter.php:96]: option_type_id seems not to be set in every case...I fixed by changing to:
`$option['option_type_id'] = (isset($option['option_type_id'])) ? (int) $option['option_type_id'] : 0;`

Comment (LorensLacis) [reply src/module-vsbridge-indexer-catalog/ArrayConverter/Product/CustomOptionConverter.php]: Ok, thank you. Added suggestion

**Review by andres-linares2211 — APPROVED** [review]

**Comment by Sproge-Kostya** [comment]
When are you planning to add this fix?

**Comment by kkdg** [comment]
@mkucmus Please merge this, this caused a lot of problems to our dev team since they didn't think it was a problem of core side.

**Comment by kkdg** [comment]
Thank you @mkucmus

**Pull request merged**